test/util/Fixtures.scala

```diff
@@ -67,7 +67,7 @@ object Fixtures {
                           |      "ip": "0.0.0.0"
                           |    }
                           |  },
-                          |  "createdAt": "2016-02-09T10:59:01.000Z",
+                          |  "createdAt": "2020-11-13T18:56:59Z",
```

Comment:
Presumably it's not just this one and we'd need to look at the other dates in here as well (e.g. the AMI)? Depends what we decide with the Prism change though. Ideally the Prism format is now stable so that we don't need to chase up fixes in other services? Let's see what DevX decide.

Reply:
Sorry, commented here before you removed it :-)

Reply:
ah yeah good shout I was being so lazy I didn't check the other fixtures!